### PR TITLE
ORCID for collaborators/contributors

### LINKED ISSUE #556: ORCID completion on contributor
The code to fetch the first and last name of a contributor based on the ORCID entered by the user is incorrectly populating the creator fields rather than the contributor fields.

This is likely a copy and paste error in the javascript.

Acceptance criteria:

- [ ] The code populates the contributor fields when an ORCID on the work Edit form additional metadata tab
- [ ] There is an automated test that show the creator is not updates and that the contributor is updated

### PR DESCRIPTION
**Pull request by hectorcorrea**
Fetches ORCID information and dumps it to the right input boxes (creator vs collaborator).

Closes #556

### CODE REVIEW

File: app/assets/javascripts/edit_utils.js [hunk 1]
```diff
@@ -279,6 +277,25 @@ $(() => {
     $(`#family_name_${suffix}`).val(familyName);
   }
 
+  // Fetch information via ORCID's public API and dumps the data into the elements indicated.
+  function fetchOrcid(orcidValue, givenNameId, familyNameId) {
+    if (isOrcid(orcidValue)) {
+      $.ajax({
+        url: `${pdc.orcid_url}/${orcidValue}`,
+        dataType: 'jsonp',
```
Comment (mccalluc) [inline app/assets/javascripts/edit_utils.js:285]: Is there a reason not to request plain JSON?

File: app/assets/javascripts/edit_utils.js [hunk 2]
```diff
@@ -279,6 +277,25 @@ $(() => {
...
+      })
+        .done((data) => {
+          const givenName = data.person.name['given-names'].value;
+          const familyName = data.person.name['family-name'].value;
+          $(givenNameId).val(givenName);
+          $(familyNameId).val(familyName);
+        })
+        .fail((XMLHttpRequest, textStatus, errorThrown) => {
+          console.log(`Error fetching ORCID for ${errorThrown}`);
```
Comment (mccalluc) [inline app/assets/javascripts/edit_utils.js:294]:
```suggestion
          console.error(`Error fetching ORCID for ${orcidValue}: ${errorThrown}`);
```

Comment (mccalluc) [reply app/assets/javascripts/edit_utils.js:294]: oh, this is copy and paste from below. Never mind; Doesn't need to change.

**Review by mccalluc — APPROVED** [review]

File: app/assets/javascripts/edit_utils.js (hunk 1 above)
Comment (hectorcorrea) [reply app/assets/javascripts/edit_utils.js:285]: Two reasons: The code already had `jsonp` (this is not a new change) and if memory serves me well, back when we implemented this we noticed that we had to use `jsonp` for the Ajax call to work, something to do with the cross-domain calls.

File: app/assets/javascripts/edit_utils.js [hunk 3]
```diff
@@ -16,9 +16,7 @@ Vite wraps it as an ESM, and there doesn't seem to be a way to specify
 an export that we can call as-needed.
 */
 
-console.log('edit_utils.js loaded...');
```
Comment (jrgriffiniii) [inline app/assets/javascripts/edit_utils.js:19]: 👍

File: app/assets/javascripts/edit_utils.js [hunk 4]
```diff
@@ -16,9 +16,7 @@ Vite wraps it as an ESM, and there doesn't seem to be a way to specify
...
 $(() => {
-  console.log('edit_utils.js hooks loading...');
```
Comment (jrgriffiniii) [inline app/assets/javascripts/edit_utils.js:21]: Perhaps also https://developer.mozilla.org/en-US/docs/Web/API/console/debug might permit this to remain?

File: spec/system/work_edit_spec.rb [hunk 5]
```diff
@@ -140,4 +140,26 @@
       expect(page.find(".activity-history-log-title", visible: false).tag_name).to eq "div"
     end
   end
+
+  context "ORCID information" do
```
Comment (jrgriffiniii) [inline spec/system/work_edit_spec.rb:144]: 👍

**Review by jrgriffiniii — APPROVED** [review]
If there are no objections to https://github.com/pulibrary/pdc_describe/pull/588/files#r1010837667, then this looks excellent to me, thank you very much @hectorcorrea !

**Pull request merged**